doc/developer-guide/api/functions/TSHttpTxnServerPush.en.rst

```diff
@@ -38,11 +38,7 @@ is not disabled by the client. You can call this API without checking whether
 Server Push is available on the transaction and it does nothing if Server Push
 is not available.
 
-
-Notes
-=====
-
-This API may be changed in the future version since it is experimental.
+This API returns an error if the URL cannot be added to the push table.
```

Comment:
I don't think this is an user friendly expression. The push table is an implementation detail, and users do not know what the table is. We could explain what the table is, but that would mean we  would have to stick with using the table (we would not be able to achieve the same thing differently).